case/src/main/kotlin/com/ritense/case_/service/CaseWidgetTabService.kt

```diff
@@ -69,6 +71,27 @@ class CaseWidgetTabService(
 
     }
 
+    @Transactional(readOnly = true)
```

Comment:
Doesn't this `readOnly` break our outbox? If you didn't already, please test this without a parent transaction (like we do in tests).